packages/xsnap/src/xsnap.c

```diff
@@ -213,7 +213,7 @@ typedef enum {
 	E_TOO_MUCH_COMPUTATION = 17,
 } ExitCode;
 
-ExitCode main(int argc, char* argv[])
+int main(int argc, char* argv[])
```

Comment:
Any particular reason this was downgraded to `int`?

Reply:
yes; @jessysaurusrex discovered that it generates a warning:

```
../../src/xsnap.c:216:1: warning: return type of 'main' is not 'int' [-Wmain-return-type]
ExitCode main(int argc, char* argv[])
```

ref https://gist.github.com/jessysaurusrex/e28c5ec0230c50c8af3adce8dee26fa5#file-gistfile1-txt-L1
https://github.com/Agoric/agoric-sdk/issues/2469#issuecomment-889429425

I tried to be clear about this in the 490cdba6de81 commit message.